src/EFCore.PG/Storage/Internal/Mapping/NpgsqlRangeTypeMapping.cs

```diff
@@ -9,21 +9,41 @@ namespace Npgsql.EntityFrameworkCore.PostgreSQL.Storage.Internal.Mapping
 {
     public class NpgsqlRangeTypeMapping : NpgsqlTypeMapping
     {
+        [NotNull] static readonly NpgsqlSqlGenerationHelper SqlGenerationHelper =
+            new NpgsqlSqlGenerationHelper(new RelationalSqlGenerationHelperDependencies());
+
+        [CanBeNull] readonly string _storeTypeSchema;
+
+        [NotNull]
+        public override string StoreType => SqlGenerationHelper.DelimitIdentifier(base.StoreType, _storeTypeSchema);
```

Comment:
I thought there was a reason I _had_ to override this for enums (this PR is just mimicking the earlier one)... something about how the base class was using the unqualified `StoreType`. 

I'll take another look. If passing it to the base works, then I agree that it would be best to do so.

(I'll also put together a follow-up PR to #605 for any changes that come from this review.)

Reply:
Yikes, not sure what I thought I was seeing back in August, but I can't find any reason for this to be overridden here.

Reverting here and will revert in the enum follow-up.